service/api/src/main/java/org/eclipse/kapua/KapuaErrorCodes.java

```diff
@@ -95,5 +95,10 @@
     /**
      * Max number of items is reached
      */
-    MAX_NUMBER_OF_ITEMS_REACHED
+    MAX_NUMBER_OF_ITEMS_REACHED,
+
+    /**
+     * User is already reserved by another connection
+     */
+    USER_ALREADY_RESERVED_BY_ANOTHER_CONNECTION
```

Comment:
Move this inside a dedicated `KapuaErrorCodes` (called ConnectionServiceErrorCodes) inside the device module